### PR TITLE
Fetch IP Address for WiFi Station

### PR DESCRIPTION
**Pull request by bhargava-cs**
- Once device is connected in STA mode start DHCP client procedure

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by bhargava-cs** [comment]
juitem

**Comment by bhargava-cs** [comment]
Hi Pillip,

Thanks for the review. I agree with you. 
In addition to that, we may keep a single application for WiFi testing i.e wm_test.

However, this patch is required  to fix a build error.
https://github.com/Samsung/TizenRT/pull/1315/commits/8126fca0dd94a13969ea29b554d26621772501c3

Please suggest whether to raise separate PR for this?

**Comment by pillip8282** [comment]
Hi Bhargava
Seperating PR would be good.

File: apps/examples/slsiwifi/slsiwifi_main.c [hunk 1]
```diff
@@ -164,6 +170,23 @@ void sw_linkUpHandler(slsi_reason_t *reason)
...
+			return;
+		}
+		if (dhcpc_request(dhcp_handle, &state) != OK) {
+			dhcpc_close(dhcp_handle);
+			return;
+		}
+		netlib_set_ipv4addr(WLAN_INTF_NAME, &state.ipaddr);
+		netlib_set_ipv4netmask(WLAN_INTF_NAME, &state.netmask);
+		netlib_set_dripv4addr(WLAN_INTF_NAME, &state.default_router);
 	} else {
```
Comment (girishks2000) [inline apps/examples/slsiwifi/slsiwifi_main.c]: This setting is static for ipv4. Restricting it to only ipv4 is not good idea. Can the test case be made compatible to work for ipv4 and v6 both. You may have to provide another wrapper (or may be already exist)

**Comment by bhargava-cs** [comment]
WM_TEST App gets IP address automatically. This change is not needed.

**Pull request closed without merging**